### ISSUE TITLE
something wrong with the example

### ISSUE THREAD

**Issue by binbinZhao2017** [stan-dev/projpred#178]
When running the command  vs <- varsel(fit) ，the error is 

**Error in eval(predvars, data, env) : object 'x1' not found**

**Comment by AlejandroCatalina** [comment]
That's right, I need to update the example on master branch (the right code is on develop). You need to run the following lines:

```
data('df_gaussian', package = 'projpred')
#data('df_binom', package = 'projpred')

split_structure <- break_up_matrix_term(y ~ x, data = df_gaussian)
df_gaussian <- split_structure$data
formula <- split_structure$formula

# fit the full model with a sparsifying prior
fit <- stan_glm(formula, family = gaussian(), data = df_gaussian,
                prior = hs(df = 1, global_scale=0.01), iter = 500, seed = 1)
#fit <- stan_glm(formula, family = binomial(), data = df_binom
#                prior = hs(df = 1, global_scale=0.01), iter = 500, seed = 1)


# perform the variable selection
vs <- varsel(fit)
```

**Comment by fweber144** [comment]
Closing this since it should be fixed in projpred 2.1.0. If not, don't hesitate to leave a comment here.
